java/runtime-native/src/main/java/org/ray/runner/RunManager.java

```diff
@@ -350,10 +346,18 @@ public void startRayProcesses() {
       startObjectStore(0, info, params.working_directory + "/store",
           params.redis_address, params.node_ip_address, params.redirect, params.cleanup);
 
+      Map<String, Double> staticResources = getResourcesFromString(params.static_resources);
+      if (!staticResources.containsKey("CPU")) {
+        staticResources.put("CPU", (double) params.num_cpus[0]);
+      }
+      if (!staticResources.containsKey("GPU")) {
+        staticResources.put("GPU", (double) params.num_gpus[0]);
+      }
```

Comment:
should we deprecate `num_cpus/num_gpus` and only use `static_resources` (at least for raylet)?

Reply:
@raulchen Can't agree more.